src/Components/Blazor/WebAssembly.Authentication/src/RemoteAuthenticatorViewCore.cs

```diff
@@ -86,10 +87,16 @@ public class RemoteAuthenticatorViewCore<TAuthenticationState> : ComponentBase w
         /// </summary>
         [Inject] public IRemoteAuthenticationService<TAuthenticationState> AuthenticationService { get; set; }
 
+        [Inject] public IRemoteAuthenticationPathsProvider RemoteApplicationPathsProvider { get; set; }
```

Comment:
I'm sure there's a good reason for this, but I haven't yet understood why this would be supplied via DI and with a public interface instead of just being a direct internal coupling to the internal `DefaultRemoteApplicationPathsProvider` class. Perhaps you can explain this bit to me later!

Reply:
Discussed offline. We moved this to the internal NS for clarity.